pkg/bundle/deploy.go

```diff
@@ -1810,6 +1810,26 @@ spec:
     namespace: CRED-SECRET-NAMESPACE
```

Comment:
If you will change the CR (names and namespaces), make sure that you will build the updated deploy.go